### PR TITLE
fix!: change default value of css to false, align with webpack

### PR DESCRIPTION
**Pull request by JSerFeng**
### Breaking Change

Align default value of css with webpack, the default value is false now.

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

**Review by LingyuCoder — DISMISSED** [review]

File: packages/rspack/src/config/defaults.ts [hunk 1]
```diff
@@ -193,7 +193,7 @@ const applyExperimentsDefaults = (
 	// IGNORE(experiments.asyncWebAssembly): The default value of `asyncWebAssembly` is determined by `futureDefaults` in webpack.
 	D(experiments, "asyncWebAssembly", false);
 	// IGNORE(experiments.css): Rspack will switch to `false` when reach 1.0 and `css-extract` is stable enough
-	D(experiments, "css", true);
+	D(experiments, "css", false);
```
Comment (SyMind) [inline packages/rspack/src/config/defaults.ts:196]: Should remove `IGNORE` comment.

**Comment by hardfist** [comment]
we should update the document about experiments.css usage at the same time

**Comment by hardfist** [comment]
this title is misleading, it should be feat or fix and the description should contain breaking change notice

File: packages/rspack-test-tools/tests/Defaults.test.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ function assertWebpackConfig(config) {
 	expect(rspackBaseConfig).toEqual(webpackBaseConfig);
 }
 
-describe("Base Defaults Snapshot", () => {
+describe.skip("Base Defaults Snapshot", () => {
```
Comment (JSerFeng) [inline packages/rspack-test-tools/tests/Defaults.test.js:72]: Wait for @SyMind to solve the conflicts

**Review by SyMind — APPROVED** [review]

**Comment by JSerFeng** [comment]
> I'll add docs asap

**Pull request merged**